symbolic-debuginfo/src/object.rs

```diff
@@ -446,7 +446,6 @@ pub enum ObjectDebugSession<'d> {
     Breakpad(BreakpadDebugSession<'d>),
     Dwarf(DwarfDebugSession<'d>),
     Pdb(PdbDebugSession<'d>),
-    Pe(PeDebugSession<'d>),
```

Comment:
Just a NOTE: removing these types is considered a breaking change.

As https://github.com/getsentry/publish/issues/1700 has been stuck and noone has tried to fix this yet, we can squeeze this breaking change into the next release still.

Please make sure to document the changes in the changelog though.

Reply:
I've updated the changelog.